engine/runtime/src/test/scala/org/enso/compiler/test/context/SuggestionBuilderTest.scala

```diff
@@ -416,6 +476,62 @@ class SuggestionBuilderTest extends CompilerTest {
       )
     }
 
+    "build type with methods, type signatures and docs" in {
+      implicit val moduleContext: ModuleContext = freshModuleContext
+      val code =
+        """type List
+          |    ## And more
+          |    type Cons
+          |    ## End
+          |    type Nil
+          |
+          |    ## a method
+          |    empty : List
+          |    empty = Nil
+          |""".stripMargin
+      val module = code.preprocessModule
+      build(code, module) should contain theSameElementsAs Seq(
+        Suggestion.Atom(
+          externalId    = None,
+          module        = "Unnamed.Test",
+          name          = "Cons",
+          arguments     = Seq(),
+          returnType    = "Cons",
+          documentation = Some(" And more")
+        ),
+        Suggestion.Atom(
+          externalId    = None,
+          module        = "Unnamed.Test",
+          name          = "Nil",
+          arguments     = Seq(),
+          returnType    = "Nil",
+          documentation = Some(" End")
+        ),
+        Suggestion.Method(
+          externalId = None,
+          module     = "Unnamed.Test",
+          name       = "empty",
+          arguments = Seq(
+            Suggestion.Argument("this", "Cons", false, false, None)
+          ),
+          selfType      = "Cons",
+          returnType    = "List",
+          documentation = Some(" a method")
+        ),
+        Suggestion.Method(
+          externalId = None,
+          module     = "Unnamed.Test",
+          name       = "empty",
+          arguments = Seq(
+            Suggestion.Argument("this", "Nil", false, false, None)
+          ),
+          selfType      = "Nil",
+          returnType    = "List",
+          documentation = Some(" a method")
+        )
+      )
+    }
+
```

Comment:
Please also add some types for local variables with type signatures. The suggestions should also work properly there too.